### DISCUSSION TITLE
[Eslint] import/no-unresolved

### DISCUSSION THREAD

**Discussion by minimal-ui-kit** [igordanchenko/yet-another-react-lightbox#33] [Q&A] [answered]
<img width="758" alt="Screen Shot 2022-10-21 at 00 41 16" src="https://user-images.githubusercontent.com/77919068/197020961-3686411b-7476-47b6-9312-41caf7f1fa70.png">

Do we have a solution for this?

Turning off `"import/no-unresolved": 0,` doesn't suit me very well in this context

The main import path seems fine. But the sub-path has problems. I use quite a few packages in my . But eslint only gives warning about this package.

**Comment by igordanchenko** [comment, marked as answer]
It looks like [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) doesn't support sub-path imports (package exports map) - https://github.com/import-js/eslint-plugin-import/issues/1810

You can add `yet-another-react-lightbox` to the ignore list until proper support for export maps is implemented in ESLint plugin.

.eslintrc.json:

```json
{
    "rules": {
        "import/no-unresolved": [
            "error",
            {
                "ignore": ["yet-another-react-lightbox/*"]
            }
        ]
    }
}
```
